addons/packages/app-toolkit/0.1.0/test/Makefile

```diff
@@ -0,0 +1,2 @@
+get-deps: ## Get all dependencies
```

Comment:
There are a few other targets that are expected. You can probably copy something like https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/antrea/1.2.3/test/Makefile